packages/cli/yaml/yaml-schema/src/schemas/ExampleResponseSchema.ts

```diff
@@ -1,9 +1,73 @@
 import { z } from "zod";
 import { ExampleTypeReferenceSchema } from "./ExampleTypeReferenceSchema";
 
-export const ExampleResponseSchema = z.strictObject({
+/**
+ * response: unknown
+ * examples:
+ *  - name: example 1
+ *    request:
+ *      stream: false
+ *    response:
+ *      body: {}
+ *
+ */
+export const ExampleBatchResponseSchema = z.strictObject({
     error: z.optional(z.string()),
     body: z.optional(ExampleTypeReferenceSchema)
 });
 
+export type ExampleBatchResponseSchema = z.infer<typeof ExampleBatchResponseSchema>;
+
+/**
+ * response-stream:
+ *   type: unknown
+ * examples:
+ *  - name: example 1
+ *    request:
+ *      stream: true
+ *    response:
+ *      stream:
+ *        - {} # event 1
+ *        - {} # event 2
+ */
+export const ExampleStreamResponseSchema = z.strictObject({
+    stream: z.array(ExampleTypeReferenceSchema)
+});
+
+export type ExampleStreamResponseSchema = z.infer<typeof ExampleStreamResponseSchema>;
+
+export const ExamplSseEventSchema = z.strictObject({
+    event: z.string(),
+    data: z.optional(ExampleTypeReferenceSchema)
```

Comment:
Can we update this `fern check` rule to check if the stream or data examples are actually valid? (https://github.com/fern-api/fern/tree/main/packages/cli/yaml/validator/src/rules/valid-example-endpoint-call)